packages/yum/groonga.spec.in

```diff
@@ -8,6 +8,7 @@
 %define _use_libstemmer %{?use_libstemmer:%{use_libstemmer}}%{!?use_libstemmer:%{_centos_ver} >= 7}
 %define _use_systemd  %{?use_systemd:%{use_systemd}}%{!?use_systemd:%{_centos_ver} >= 7}
 %define _use_rapidjson %{?use_rapidjson:%{use_rapidjson}}%{!?use_rapidjson:%{_centos_ver} == 7}
+%define _use_bundled_rapidjson %{?use_bundled_rapidjson:%{use_bundled_rapidjson}}%{!?use_bundled_rapidjson:%{_centos_ver} > 7}
```

Comment:
In my opinion, this condition is " >= 8" better than "> 7". 
Because ">= 8" make the meaning of can use from centos 8 clear.

Reply:
Thanks!
I've fixed and added commit comment for that.